runtime/vm/swalk.c

```diff
@@ -481,6 +481,13 @@ UDATA walkFrame(J9StackWalkState * walkState)
 			return J9_STACKWALK_KEEP_ITERATING;
 		}
 
+		/* Skip methods with JvmtiMountTransition annotation */
+		if (J9_ARE_ALL_BITS_SET(walkState->flags, J9_STACKWALK_SKIP_JVMTI_MOUNT_TRANSITION_FRAMES)
+			&& J9_IS_JVMTI_MOUNT_TRANSITION(walkState->method)
+		) {
+			return J9_STACKWALK_KEEP_ITERATING;
+		}
```

Comment:
To be consistent with https://github.com/eclipse-openj9/openj9/pull/16654, the code changes in this PR should be wrapped with `JAVA_SPEC_VERSION >= 20`.
```suggestion
#if JAVA_SPEC_VERSION >= 20
		/* Skip methods with JvmtiMountTransition annotation */
		if (J9_ARE_ALL_BITS_SET(walkState->flags, J9_STACKWALK_SKIP_JVMTI_MOUNT_TRANSITION_FRAMES)
			&& J9_IS_JVMTI_MOUNT_TRANSITION(walkState->method)
		) {
			return J9_STACKWALK_KEEP_ITERATING;
		}
#endif /* JAVA_SPEC_VERSION >= 20 */
```